.github/CODEOWNERS

```diff
@@ -1 +1,17 @@
 * @nsmithtt
+*.fbs @tapspatel
+/.github/ @vmilosevic
+/include/ttmlir/Conversion/TTNNToEmitC/ @svuckovicTT
+/include/ttmlir/Dialect/TT/ @sdjordjevicTT @rpavlovicTT @mrakitaTT
+/include/ttmlir/Dialect/TTIR/ @sdjordjevicTT @rpavlovicTT @mrakitaTT
+/include/ttmlir/Dialect/TTIR/Analysis/ @nobradovictt
```

Comment:
I would like to share ownership of transforms with @sdjordjevicTT / you / whoever wants it as well.

Reply:
I'll just list you as part of the directory above which will capture all IR/Passes/Transforms/Analysis changes.